_layouts/prototype.html

```diff
@@ -3,39 +3,66 @@
 ---
 
 <div class="prototype">
-  <h2>{{ page.title }}</h2>
-  <p>{{ page.description }}</p>
-  
-  {% if page.prototype_url %}
-    <p><a href="{{ page.prototype_url }}">Try it now!</a></p>
-  {% endif %}
+  <div class="container">
+    <div class="col1">
+      <div class="row">
+        <p>
+          {{ page.category }} 
+          
+          {% if page.license == "MIT" or page.license == "mit" %}
 
-  {% if page.vimeo %}
-    <h3>Demo:</h3>
-    
-    <div class="resp-container">
-      <iframe class="resp-iframe" src="{{ page.vimeo }}" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe>
+          <a href="{{ page.repo_url }}">
+            <img src="/images/mit-badge.svg" height="20" />
+          </a>
+          {% endif %}
+        </p>
+      </div>
+      
+      <h2>{{ page.title }}</h2>
+      <p>{{ page.description }}</p>
+      <div class="row">
+        {% if page.prototype_url %}
+        <a class="btn" href="{{ page.prototype_url }}">Try it!</a>
+        {% endif %} {% if page.repo_url %}
+        <a class="btn" href="{{ page.repo_url }}">Github Repo</a>
+        {% endif %}
+      </div>
     </div>
-  {% elsif page.demo %}
-    <h3>Demo:</h3>
-    <video controls>
-      <source src="{{ page.demo }} " type="video/mp4">
-    </video>
-  {% elsif page.screenshot %}
-    <img src="{{ page.screenshot }}" />
-  {% endif %}
-
-  {% if page.repo_url %}
-    <div>
-      <h3>Repo:</h3><a href="{{ page.repo_url }}">{{ page.repo_url }}</a>
+    <div class="col2">
+      <div class="borderText">
+        <div class="verticalText">
+          <a href="{{ site.url }}" class="verticalText">Return to all projects</a>
+          <div class="outerborder"></div>
+        </div>
+      </div>
     </div>
-  {% endif %}
+  </div>
+    <section>
+      {% if page.vimeo %}
```

Comment:
Let's make the the vimeo and demo video containers smaller as they get to be too large on larger browser sizes.